webui/react/src/components/kit/LineChart.tsx

```diff
@@ -1,6 +1,5 @@
-import React, { useMemo, useState } from 'react';
-import AutoSizer from 'react-virtualized-auto-sizer';
```

Comment:
Note that dropping this = you can remove `react-virtualized-auto-sizer` and `@types/react-virtualized-auto-sizer` from package.json